core/web/resolver/mutation.go

```diff
@@ -592,3 +594,63 @@ func (r *Resolver) executeJobProposalAction(ctx context.Context, action jobPropo
 
 	return jp, nil
 }
+
+func (r *Resolver) SetServicesLogLevels(ctx context.Context, args struct {
+	Input struct{ Config *LogLevelConfig }
+}) (*SetServicesLogLevelsPayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	fmt.Println(*args.Input.Config)
```

Comment:
Can remove this

Reply:
Also, should you check that `Input.Config != nil`? Seems like since it's a pointer it might be optional? Not sure why it'd be a pointer though, it seems to be non-nullable in the GQL schema.

Reply:
True, it shouldn't be a pointer, I removed that.